src/types/controlPlaneConfig.ts

```diff
@@ -8,7 +8,29 @@ export type DestinationDefinition = {
   Config: Record<string, unknown>;
 };
 
-export type Destination<DestinationConfig = Record<string, unknown>> = {
+export type AccountDefinitionConfig = { refreshOAuthToken?: string };
+
+export type AccountDefinition<Config = AccountDefinitionConfig> = {
+  Name: string;
+  Config: Config;
+  AuthenticationType: string;
+};
+export type Account<
+  OptionsT = Record<string, unknown>,
+  SecretT = Record<string, unknown>,
+  AccountDefinitionT = AccountDefinition,
+> = {
+  ID: string;
+  Options?: OptionsT;
```

Comment:
Options can't be optional. If there is not anything inside object it would be empty object.